dev-docs/multipart-subscriptions-protocol.md

```diff
@@ -27,6 +26,9 @@ Content-Type: application/json
 --graphql--
 ```
 
+> Note: because the parts are always JSON, it is never possible for `\r\n--graphql` to appear in the contents of a part. For convenience, servers MAY use `graphql` as a boundary.
```

Comment:
is that warning necessary? Boundary parsing is standard in multipart

Reply:
I added this as a non-normative note for implementers more than a warning. This saves the time to implementers trying to think of a correct boundary. I'm fine skipping it if we think it doesn't have value.